QtCollider/widgets/image_painter.h

```diff
@@ -144,7 +144,6 @@ struct ImagePainter
         bool tileVertically = verticalMode == TileVertically;
         bool tileHorizontally = horizontalMode == TileHorizontally;
 
-        qreal x_origin = rect.left();
```

Comment:
to me, this would read better if you kept it and used it:

```
  if (tileHorizontally) {
                rect.moveTop( y_origin );
                rect.moveLeft( x_origin + rect.width() );
            }
```